app/models/live_result.rb

```diff
@@ -42,7 +45,56 @@ def complete?
 
   private
 
+    def should_recompute?
+      saved_change_to_best? || saved_change_to_average?
+    end
+
     def notify_users
       ActionCable.server.broadcast("results_#{round_id}", serializable_hash)
     end
+
+    def recompute_advancing
+      round_results = LiveResult.where(round: round)
+      round_results.update_all(advancing: false)
+
+      missing_attempts = round.total_registrations - round_results.count
+      puts(round.registrations.inspect)
```

Comment:
:smiling_imp:

Reply:
me trying to debug the issue before I added another expect